zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchSpanConsumer.java

```diff
@@ -46,13 +46,13 @@
     this.indexTypeDelimiter = es.indexTypeDelimiter();
     this.searchEnabled = es.searchEnabled();
     this.delayLimiter = DelayLimiter.newBuilder()
-      .ttl(es.autocompleteTtl())
+      .ttl(es.autocompleteTtl(), TimeUnit.MILLISECONDS)
```

Comment:
this parameter changed because the units are different. Ex the index mutes are second granularity